go.mod

```diff
@@ -81,13 +81,13 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
```

Comment:
As I understand it, these changes are being performed to correctly apply the following version bump attempt by dependabot

https://github.com/openshift/assisted-service/pull/6747/files

The changes here make sense, I can see that you have bumped the package versions to the highest possible versions while still remaining on the same go version.

I also see some general tidy up such as the addition of the `-s` (silent) flag to curl operations and the formal download of the correct oc version for the architecture. All of these are sensible changes but I think we should mention that we are doing both of these things here.

Reply:
Roger, 
This reason I changed the client download was that some time the url `https://dl.k8s.io/release/stable.txt` was returning code  500 sometimes. 

+  it's not containing the `oc` tool which is include in the other link